src/sdkToEventsApiConverter.ts

```diff
@@ -56,6 +56,13 @@ export function convertEvents(
         currentConsentState = user.getConsentState();
     }
 
+    // TODO: Apply this new attribute to event-models
```

Comment:
I approved https://github.com/mParticle/js-models/pull/119, so I think we should release that one and then this can be removed once the package.json is updated.